packages/notifications/DrawerResourceGetDrawerEntries/index.ts

```diff
@@ -0,0 +1,161 @@
+// @ts-ignore
```

Comment:
@johnsonm325 did you intend to change the notifications client? It's possible to only generate and build 1 client with NX (I see @catastrophe-brandon updated the test - but just want to make sure this was intended and we didn't generate/build all the clients).

Reply:
I did not intend to generate/build all of the clients.

Maybe let me take another stab at this. I was having a hard time with the steps for generating.

Reply:
@johnsonm325 I still see the modified `notifications-client` in this commit. Are we still intending to update the notifications-client too? Or just `vulnerabilities`?

Reply:
@johnsonm325 I opened an alternative PR that just updates the vulnerabilities client and the failing notifications test here: https://github.com/RedHatInsights/javascript-clients/pull/368.